src/App.vue

```diff
@@ -1,7 +1,7 @@
 <template>
     <div id="app">
         <SignIn v-if="this.$route.path === '/'" />
-        <div v-else-if="this.$route.params.currentWorkspace || this.$route.path === '/user'">
+        <div v-else-if="this.$route.params.currentWorkspace || this.$route.path === '/user' || this.$route.path === '/user-management'">
```

Comment:
Would it be easier to put the management UI under `/user/management`?

Reply:
@perigrin That's a good idea. I've been working on Auth tokens, and since both User Management and Auth tokens are admin-only features, I was thinking of moving both to separate tabs under `/admin`. Does that make sense to you, or would rather see something like `/user/management` and `/user/authentication-tokens` (or `/user/tokens`)?

Reply:
My gut reaction says "that works too" with `/admin/users` and `/admin/tokens`.